{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/utils/logging_utils.py

```diff
@@ -3,6 +3,7 @@
 import os
 import socket
 
+from pip._internal.operations import freeze
```

Comment:
pip freeze within python? :boom: of course thats a thing. Good idea putting it in the logs!